graphql_api/types/bundle_analysis/base.py

```diff
@@ -295,10 +295,18 @@ def resolve_bundle_report_measurements(
     )
 
     # All measureable names we need to fetch to compute the requested asset types
-    if not asset_types or ASSET_TYPE_UNKNOWN in asset_types:
+    if not asset_types:
         measurables_to_fetch = [
             item for item in list(BundleAnalysisMeasurementsAssetType)
         ]
+    elif ASSET_TYPE_UNKNOWN in asset_types:
+        measurables_to_fetch = [
+            BundleAnalysisMeasurementsAssetType.REPORT_SIZE,
```

Comment:
I saw ASSET_SIZE is another variable on this type; jw why its not included in this list?

Reply:
ASSET_SIZE is for tracking the size of individual javascript assets, so the sum of all asset measurements equals to the JAVASCRIPT_SIZE. And all the other types are aggregate types, notably REPORT_SIZE is the sum of all assets. 
So we can compute the trends line of unknown asset sizes by doing a computation of:
`REPORT_SIZE = JS_SIZE - STYLESHEET_SIZE - FONT_SIZE - IMAGE_SIZE`
and
ASSET_SIZE can be ignored because `JS_SIZE = sum(ASSET_SIZE)`

Reply:
ahhhh okay makes a lot of sense! Thanks